pkg/analyze/archivereader_test.go

```diff
@@ -0,0 +1,601 @@
+package analyze
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"k8s.io/klog/v2"
+	"reflect"
+	"sort"
+	"strings"
+	"testing"
+	"testing/fstest"
+
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+)
+
+func defaultPodYaml(name string) []byte {
+	return []byte(strings.TrimPrefix(`
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: scylla
+  name: `, "\n") + name + "\n")
+}
+
+func defaultPodObject(name string) *corev1.Pod {
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: "scylla",
+		},
+	}
+}
+
+func SortByName(objs []interface{}) {
```

Comment:
doesn't need to be exported from test suite
```suggestion
func sortByName(objs []interface{}) {
```

Reply:
done